Core/src/org/sleuthkit/autopsy/logicalimager/dsp/LogicalImagerDSProcessor.java

```diff
@@ -207,12 +221,14 @@ private void run(String deviceId, String timeZone,
     ) throws NoCurrentCaseException {
         addLogicalImageTask = new AddLogicalImageTask(deviceId, timeZone, src, dest,
                 progressMonitor, callback);
-        new Thread(addLogicalImageTask).start();
+        thread = new Thread(addLogicalImageTask);
+        thread.start();
     }
 
     @Override
     public void cancel() {
         if (addLogicalImageTask != null) {
+            thread.interrupt();
```

Comment:
Is this still necessary? And saving the thread in general?

Reply:
Yes. When I removed the interrupt, the file copying (non-VHD case) was left with an open file, so it was not able to delete the destination directory.